person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_reader :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] Kindly make this method private, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Private%20method%20of_age%3F) requirement.